internal/registry/service.go

```diff
@@ -228,14 +230,9 @@ func (client *Client) UpdateBuild(
 	return resp.Payload.Build.ID, nil
 }
 
-// GetIterationFromChannel loads the iterationId associated with a current channel. If the
+// GetChannel loads the iterationId associated with a current channel. If the
```

Comment:
```suggestion
// GetChannel loads the named channel that is associated to the bucket slug . If the
```

Reply:
The latest version still boasts the old documentation, is it voluntary? I'd agree with Wilken's suggestion

Reply:
Oh no, I forgot to commit these changes! Thanks